tests/test_unit.py

```diff
@@ -713,24 +713,20 @@ def test_extract_header_from_markdown_alternate_header(self):
 
     def test_extract_header_from_markdown_bad_headers(self):
```

Comment:
Seems like this one should also be parameterized.

Reply:
Turned relevant tests to be parameterized.